optimade/validator/validator.py

```diff
@@ -790,7 +795,7 @@ def _construct_single_property_filters(
                 if query_optional:
                     return (
                         None,
-                        "Optional query {query!r} returned {reversed_response.status_code}.",
+                        "Optional query {query!r} returned {response.status_code}.",
```

Comment:
How can you retrieve the `status_code` if the `response` tests as `False` in this code branch?

Reply:
Huh, you're right, whilst this typo still stands, it's not what was causing the issue I ran into... will fix

Reply:
Well it might be `bool(response)` return `True`/`False` for some status codes of the response? I'm not sure...